docs/main_text.md

```diff
@@ -60,31 +60,150 @@ If you would prefer to report issues via e-mail, you can also email [ccdl@alexsl
+allowing for one-to-many mappings.
+
+TODO: Filtering for probe quality?
 
 ## Aggregations
 
+refine.bio allows users to aggregate their selected samples in two ways: by experiment or by species.
+We use the term aggregate or aggregation to refer to the process of combining _individual samples_ to form a _multi-sample_ gene expression matrix (see also: [Downloadable Files](#downloadable-files)).
+
+* **By experiment:** Samples that belong to the same experiment will become a single gene expression matrix. 
+If you have selected all samples from two experiments with 10 and 15 samples, respectively, and have chosen the `by experiment` option, you will receive two gene expression matrices with 10 and 15 samples, respectively.
+
+* **By species:** All samples assaying the same species will be aggregated into a single gene expression matrix.
+If you have selected three experiments each from human and mouse and the `by species` option, you receive two gene expression matrices that contain all human and all mouse samples, respectively.
+
+For either aggregation method, we summarize duplicate Ensembl gene IDs to the mean expression value and only include genes (rows) that are represented in **all** samples being aggregated.
+Note that some early generation microarrays measure fewer genes than their more recent counterparts, so their inclusion when aggregating `by species` may result in a small number of genes being returned. 
+The aggregation methodology for species compendia is different; see [Species compendia](#species-compendia) for more information.
+
+TODO: screenshots? illustrations? at least one of the inner join
+
 ## Transformations
 
 ### Quantile normalization
 
-### Gene transformations
+TODO: Linking to QN [issue](https://github.com/AlexsLemonade/refinebio/issues/488) and [PR](https://github.com/AlexsLemonade/refinebio/pull/519) for now.
 
+### Gene transformations
 
-# CCDL Developed Packages
-
-## Illumina SCAN
+In some cases, it may be useful to row-normalize or transform the gene expression values in a matrix (e.g., following aggregation and quantile normalization). 
+We offer the following options for transformations:
 
-## Identifier Refinery
+* **None:** No row-wise transformation is performed.
+* **Z-score:** Row values are [z-scored](https://en.wikipedia.org/wiki/Standard_score) using the [`StandardScaler`](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html) from [`scikit-learn`](http://scikit-learn.org/stable/index.html).
+* **Zero to one:** Rows are scaled to values `[0,1]` using the [`MinMaxScaler`](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html) from [`scikit-learn`](http://scikit-learn.org/stable/index.html).
+We expect this transformation to be most useful for certain machine learning applications (e.g., those using cross-entropy as a loss function).
 
+TODO: screenshots? plots that show effect of different scaling options?
```

Comment:
If we have use cases where certain transformations are applied, we could link them here. 
Is there anything we could add to None and Z-score which would help users aren't familiar to choose?

Reply:
They use z-scores in some TCGA data (search `z-score` on [this page](http://www.cbioportal.org/faq.jsp)). I think that would be a good example to mention, perhaps in the bullet point above.

Reply:
Yes, that would be helpful. 

```This measure is useful to determine whether a gene is up- or down-regulated relative to the normal samples or all other tumor samples.``` 👌